paperqa/contrib/openreview_paper_helper.py

```diff
@@ -0,0 +1,152 @@
+import json
+import logging
+import os
+from pathlib import Path
+from typing import Any, Dict, List, Optional
+
+import openreview
+import requests
```

Comment:
We avoid `requests` as it's not `async`, can you move to `httpx` (preference) or `aiohttp` (what we use currently, but want to move away from)